astropy/wcs/tests/test_wcs.py

```diff
@@ -1508,3 +1508,15 @@ def test_distortion_header(tmpdir):
         w2 = wcs.WCS(hdulist[0].header)
 
     assert w.pixel_to_world(*cen).separation(w2.pixel_to_world(*(siz / 2))) < 1.e-3 * u.mas
+
+
+def test_chandra_pixlist_wcs():
+    hdr_file = get_pkg_data_filename('data/chandra-pixlist-wcs.hdr')
+    hdr = fits.Header.fromtextfile(hdr_file)
+    with pytest.warns(wcs.FITSFixedWarning):
+        w = wcs.WCS(hdr, keysel=['image', 'pixel'], colsel=[11, 12])
+    assert w.naxis == 2
+    assert list(w.wcs.ctype) == ['RA---TAN', 'DEC--TAN']
+    assert np.allclose(w.wcs.crval, [229.38051931869, -58.81108068885])
+    assert np.allclose(w.wcs.pc, [[1, 0], [0, 1]])
+    assert np.allclose(w.wcs.cdelt, [-0.00013666666666666, 0.00013666666666666])
```

Comment:
```suggestion
    assert np.allclose(w.wcs.cdelt, [-0.00013666666666666, 0.00013666666666666])
    assert np.close(w.wcs.lonpole, 180.)
```
I suggest also testing that in fact only the lonpole for that specific coordinate system is read. If that's not the case, that's  WCSLIB bug, so I'm OK if you reject this suggestion as out-of-scope, but that's the use case that this started with #8389 and #7180. And, I expect, it's probably the most relevant use case, since X-ray astronomy is the only field I know of that uses tables with multiple WCS as their default format.